### ISSUE TITLE
any future plans for touch(swipe, pinch, multi touch,etc)

### ISSUE THREAD

**Issue by mashrit** [tio/input-emulator#5]

**Comment by lundmar** [comment]
Yes. However, for version 1.0 the priority is to add keyboard language/model configuration support to support unicode input strings. Of course, you are welcome to try add it yourself if you have a bit of coding experience.
